.github/workflows/publish-marketplace.yml

```diff
@@ -0,0 +1,28 @@
+name: Automatic Publish to Marketplace
+
+on:
+  workflow_run:
+    workflows: ["Automatic Release"]
+    branches: [master]
+    types: 
+      - completed
+  workflow_dispatch
+
+env:
+  PAT: ${{ secrets.PAT }}
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+  PRODUCTION: ${{ secrets.PRODUCTION }}
+  OPEN_VSX_ACCESS_TOKEN: ${{ secrets.OPEN_VSX_ACCESS_TOKEN }}
```

Comment:
i think this env variable would not be needed now it's seperated into two yaml files. same with `PAT` in the open-vsx action

Reply:
yes, good point, removed those 👍